cmd/internal/server/core.go

```diff
@@ -113,19 +135,21 @@ func (s *Server) Start() error {
 	}
 	s.fuse = controlChannels{stopCh: fuseServerStopCh, stoppedCh: fuseServerStoppedCh}
 
-	if s.opts.CPUProfilePath != "" {
-		f, err := os.Create(s.opts.CPUProfilePath)
-		if err != nil {
-			log.Fatal(err)
+	if !s.forVerifyInstall {
+		if s.opts.CPUProfilePath != "" {
```

Comment:
I think we should leave the CPUProfilePath stuff in. It might be interesting to test just that portion of the server, and there's no harm in having it still work.

Reply:
Deferring that to when we need it, since `verify-install` (currently) has no way to set the CPU profile.